tool/project-setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DATA_DIR=~/dust/data
+INST_DIR=~/maiden
+
+# setup source and catalog dirs
+mkdir -pv $DATA_DIR/sources
+mkdir -pv $DATA_DIR/catalogs
+
+# copy initial contents; ignoring failures
+set +e
+cp -v $INST_DIR/dist/sources/*.json $DATA_DIR/sources/
```

Comment:
These use `$INST_DIR` instead of `$DATA_DIR` and logged the following error when running from `~/maiden` (maybe all of the above is expected?):
```
cp: cannot stat '/home/we/maiden/dist/sources/*.json': No such file or directory
cp: cannot stat '/home/we/maiden/dist/catalogs/*.json': No such file or directory
```

Reply:
It seems like the catalog files are empty too. Very possible I missed a step somewhere...

Reply:
The catalog dir should be empty; source shouldn't. Can you verify that you did a `make release` for current master - there was some stuff moved around / added once the catalogs files were moved into their final location on GH.

Reply:
Yep, I just tried it again. I don't know what I'm doing wrong. The head is at 2baca99. I'm consistently getting that same error above. I have been running `project-setup.sh` `ssh`ed into the norns; I tried running it from the source directory on my dev machine and that didn't work either. 

Perhaps I'm missing a dependency somewhere else that somehow isn't surfacing until I run that script?